mmv1/products/cloudrunv2/Job.yaml

```diff
@@ -487,6 +487,25 @@ properties:
                       name: 'readOnly'
                       description: |-
                         If true, mount this volume as read-only in all mounts. If false, mount this volume as read-write.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'nfs'
+                  description: |-
+                    NFS share mounted as a volume. This feature requires the launch stage to be set to ALPHA or BETA.
+                  min_version: beta
```

Comment:
Add exactly_one_of?

I wonder why exactly_one_of in other fields are commented out.

Reply:
I can add it, but I'm not sure why it's there and commented out. It looks like it's been commented out since the very first commit so there's no context in the commit history.

Reply:
Not sure whether it's necessary since in our API these fields are grouped into an oneof. I'll leave TF reviewer to make a suggestion.

Reply:
I think it is commented out to show the intention, but the validation does not work for nested fields. I would probably suggest adding a similar comment (and updating the others) for consistency/clarity, but it isn't strictly needed for anything.

Reply:
ok, added the comment